src/test/groovy/com/mechanitis/demo/spock/OrderingServiceSpec.groovy

```diff
@@ -0,0 +1,125 @@
+package com.mechanitis.demo.spock
+
+import com.synacy.gradprogram.spock.exercise.Cart
+import com.synacy.gradprogram.spock.exercise.Item
+import com.synacy.gradprogram.spock.exercise.ItemType
+import com.synacy.gradprogram.spock.exercise.Order
+import com.synacy.gradprogram.spock.exercise.OrderStatus
+import com.synacy.gradprogram.spock.exercise.OrderingService
+import com.synacy.gradprogram.spock.exercise.UnableToCreateOrderException
+import spock.lang.Specification
+
+class OrderingServiceSpec extends Specification {
+
+    OrderingService service
+    Cart cart
+    def items = [
+            new Item ("Rice", 50.00, ItemType.FOOD),
+            new Item ("Chicken", 20.00, ItemType.FOOD),
+            new Item ("Nasa", 34.5, ItemType.CLOTHING),
+            new Item ("Air-con", 34.5, ItemType.APPLIANCE),
+            new Item ("Laptop", 34.5, ItemType.GADGET),
+            new Item ("Nike", 500.54, ItemType.CLOTHING)
+    ]
+//    Item item1 = new Item ("Rice", 50.00, ItemType.FOOD)
+//    Item item2 = new Item ("Chicken", 20.00, ItemType.FOOD)
+//    Item item3 = new Item ("Nasa", 34.5, ItemType.CLOTHING)
+//    Item item4 = new Item ("Air-con", 34.5, ItemType.APPLIANCE)
+//    Item item5 = new Item ("Laptop", 34.5, ItemType.GADGET)
+
+
+    def setup() {
+        service = new OrderingService()
+    }
+
+    def "should return true when cart contains food item type"() {
+        when:
+        boolean containsFoodType = service.cartContainsFoodItem(cart = new Cart(UUID.randomUUID(), items))
+
+        then:
+        containsFoodType
```

Comment:
This is okay but it would be easier to read if we specify what we are expecting like this:
```
true == containsFoodItem
```

This should apply to the rest of the boolean assertions.

Reply:
Also we need to have a test when the scenario is the Cart does not have any food item.